skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -638,4 +638,28 @@ static function onArticleSaveComplete( &$article, &$user, $text, $summary,
 		$wgMemc->delete( wfMemcKey( 'mOasisRecentRevisions2', $article->getTitle()->getArticleId() ) );
 		return true;
 	}
+
+	private function renderTemplateSubtitle() {
+		global $wgUser;
```

Comment:
You do not need to use globals in a context of a controller. Just go with `$this->getContext()->getUser()`.